src/MetricsUploader/CaptureMetricTest.cpp

```diff
@@ -176,4 +178,51 @@ TEST(CaptureMetric, SendCaptureSucceededWithoutCompleteData) {
   EXPECT_TRUE(metric.SendCaptureSucceeded(std::chrono::milliseconds{5}));
 }
 
+TEST(CaptureMetric, SendCaptureWithFile) {
+  MockUploader uploader{};
+
+  EXPECT_CALL(uploader, SendCaptureEvent(_, _))
+      .Times(1)
+      .WillOnce(
+          [](const OrbitCaptureData& capture_data, OrbitLogEvent_StatusCode status_code) -> bool {
+            EXPECT_EQ(status_code, OrbitLogEvent_StatusCode_SUCCESS);
+            EXPECT_EQ(capture_data.duration_in_milliseconds(), 5);
+            EXPECT_TRUE(HasSameCaptureStartData(capture_data, kTestStartData));
+            EXPECT_EQ(capture_data.file_size(), 10);
```

Comment:
Nit: I prefer to add a constant `kTestCaptureFileSize` to mention that the test file only have 10 bytes; otherwise I might have to go to the test file to check that the file size is actually 10.

Reply:
Or as what you are doing in SendCaptureWithoutFile, add a comment after this line to mention that the test file has only 10 bytes.

Reply:
done